src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -0,0 +1,232 @@
+        />
+      </template>
+    </DetailsItem>
+
+    <DetailsItem
+      expandable
+      :label="$t('transaction.advancedDetails')"
+    >
+      <DetailsItem :label="$t('common.nonce')">
+        <!-- TODO -->
+        TODO
+      </DetailsItem>
+      <DetailsItem :label="$t('common.callData')">
+        <!-- TODO -->
+        TODO
+      </DetailsItem>
+      <DetailsItem
+        :label="$t('common.contractId')"
+        small
+      >
+        <!-- TODO -->
+        <AddressFormatted
+          :address="'TO_DOTODOTODO'"
+        />
+      </DetailsItem>
+    </DetailsItem>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  PropType,
+  ref,
+} from '@vue/composition-api';
+import BigNumber from 'bignumber.js';
+import { IAccount, ICreateMultisigAccount } from '../../types';
+import { AETERNITY_SYMBOL, getAccountNameToDisplay, MAGNITUDE } from '../utils';
+import { useAccounts, useMultisigAccounts } from '../../composables';
+import { useGetter } from '../../composables/vuex';
+
+import Avatar from './Avatar.vue';
+import AddressTruncated from './AddressTruncated.vue';
+import AddressFormatted from './AddressFormatted.vue';
+import AccountItem from './AccountItem.vue';
+import BtnPill from './buttons/BtnPill.vue';
+import DetailsItem from './DetailsItem.vue';
+import DialogBox from './DialogBox.vue';
+import TokenAmount from './TokenAmount.vue';
+import FormSelect, { FormSelectOption } from './form/FormSelect.vue';
+
+import PendingIcon from '../../icons/animated-pending.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    DetailsItem,
+    AccountItem,
+    DialogBox,
+    TokenAmount,
+    BtnPill,
+    Avatar,
+    AddressTruncated,
+    FormSelect,
+    PendingIcon,
+    AddressFormatted,
+  },
+  props: {
+    signers: { type: Array as PropType<ICreateMultisigAccount[]>, required: true },
+    confirmationsRequired: { type: Number, required: true },
+  },
+  setup(props, { root }) {
+    const {
+      isLocalAccountAddress,
+    } = useAccounts({ store: root.$store });
+
+    const {
+      estimateMultisigAccountDeployGasFee,
+    } = useMultisigAccounts({ store: root.$store });
+
+    const loading = ref(false);
+    const gasFee = ref(0);
+    const accounts = useGetter<IAccount[]>('accounts');
+    const accountsOptions = computed(
+      (): FormSelectOption[] => accounts.value.map((acc) => ({
+        text: getAccountNameToDisplay(acc),
+        value: acc.address,
+        address: acc.address,
+      })),
+    );
+    const creatorAddress = ref<string>(accounts.value[0].address);
+    const creatorAccount = computed(
+      () => accounts.value.find(({ address }) => address === creatorAddress.value),
+    );
+
+    (async () => {
+      loading.value = true;
+      const rawGasFee = await estimateMultisigAccountDeployGasFee(
+        null,
+        props.confirmationsRequired,
+        [...props.signers.map(({ address }) => address)],
```

Comment:
```suggestion
        props.signers.map(({ address }) => address),
```

Reply:
Hahaha, that's a good one :) Totally right... fixing this.